### PR TITLE
Convert hocr-merge-dc to use lxml, #8, add help #13

### PR DESCRIPTION
**Pull request by zuphilip**
This is the first attempt to switch the code to `lxml`. It seems not to produce any error, but also it does not really do anything, I think...

### CODE REVIEW

**Comment by kba** [comment]
You could probably test this works as expected with the sample data in `./tests`.

**Comment by zuphilip** [comment]
Yes, that is what I am doing, but the result looks like nothing happend:
```
zuphilip@zuphilip-VirtualBox:/etc/hocr-tools$ ./hocr-merge-dc tests/dcsample2.xml tests/sample.html
<html>
<head>
    <meta name="ocr-id" value="OCRopus Revision: 312"/>
    <meta name="ocr-recognized" value="lines text"/>
    <meta name="DC.creator" value="James T. Kirk"/>
    <meta name="DC.title" value="Nomad"/>
    <meta name="DC.publisher" value="NASA"/>
</head>
<body>
...
```

File: hocr-merge-dc [hunk 1]
```diff
@@ -15,46 +17,57 @@ dcknown = [
...
 
 ### remove all existing META tags representing Dublin Core metadata
 
-hocr_meta = Evaluate("//HEAD",hocr_doc)
+hocr_meta = hocr_doc.xpath("//HEAD|//head");
 assert hocr_meta!=[]
 hocr_meta = hocr_meta[0]
 
-hocr_nodes = Evaluate("//HEAD//META[starts-with(@name,'DC.')]",hocr_doc)
+hocr_nodes = hocr_doc.xpath("//HEAD//META[starts-with(@name,'DC.')]")
```
Comment (zuphilip) [inline hocr-merge-dc]: Uppercase/lowercase matters here for `xpath`. That is the problem...

**Comment by zuphilip** [comment]
Okay, now this is working again. @kba Any thoughts here? Or are we good with this PR?

**Comment by kba** [comment]
I've added a basic test and it works. All unit tests pass now :dancer: https://travis-ci.org/UB-Mannheim/hocr-tools/builds/131938912

**Pull request merged**